zerver/tests/test_home.py

```diff
@@ -439,7 +439,7 @@ def test_num_queries_for_realm_admin(self) -> None:
         # Verify number of queries for Realm admin isn't much higher than for normal users.
         self.login("iago")
         flush_per_request_caches()
-        with self.assert_database_query_count(48):
+        with self.assert_database_query_count(50):
```

Comment:
I'm a bit puzzled that this added two database queries, not 1; any idea why that is?

(We have these tests to detect database query in loop situations, and I've had kinda thought that the scheduled message stuff only did a single bulk database query to fetch everything).